### PR TITLE
Release EXOTIC 4.2.0: Plotting enhancements, data precision improvements and API modernization

### PR DESCRIPTION
**Pull request by jpl-jengelke**
- Fix scales on FOV plots and axial range setting 
  - Set circular reticule to red when aperture photometry is selected 
  - Add `opt_method`, `aper` and `annulus` labels to legend 
- Properly scale bound for host star isolation in plate solutions 
  - Account for J2000 RA and DEC 
  - Bound user coordinates for accuracy (limit 5 point spread function[PSF]) 
  - Update conditions for using PSF versus aperture identification
- Stability improvements in querying and analyzing known archive priors 
- Dependency updates to support Python 3.12+
- Various enhancements, bug fixes and stability improvements

### CODE REVIEW

File: exotic/python
Comment (rzellem) [inline exotic/python:1]: ...do we need this empty file?

**Review by rzellem — CHANGES_REQUESTED** [review]
Do we need the empty python file: `exotic/python`?

File: exotic/python
Comment (jpl-jengelke) [reply exotic/python:1]: No, I don't believe we do ... :)

**Review by rzellem — APPROVED** [review]

File: exotic/python
Comment (jpl-jengelke) [reply exotic/python:1]: Done in #1325 ...

**Pull request merged**